Tasks/PublishTestResultsV2/publishtestresults.ts

```diff
@@ -165,11 +172,37 @@ async function run() {
             }
         }
         tl.setResult(tl.TaskResult.Succeeded, '');
-    } catch (err) {       
+    } catch (err) {
         tl.setResult(tl.TaskResult.Failed, err);
     } finally {
         ci.fireConsolidatedCi();
     }
 }
 
+function readAndPublishTestRunSummaryToEvidenceStore(testRunner: string) {
+    try {
+        const agentVersion = tl.getVariable('Agent.Version');
+        if(semver.lt(agentVersion, "2.162.1")) {
+            throw "Required agent version greater than or equal to 2.162.0";
```

Comment:
Don't throw error. just log it as unable to push metadata..

Reply:
there's a catch below that doesn't throw it, it just logs it.